dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -25,6 +25,7 @@ pub contract DSSCollection: NonFungibleToken {
         id: UInt64,
         name: String,
         description: String,
+        productName: String,
```

Comment:
If we do a dapper wide one, would product just be dapper?  This can be implied from the type name in the slot anyways.  Not super against it though.

Reply:
Yes. We could do that.